projects/Mallard/src/hooks/use-edition-provider.tsx

```diff
@@ -20,6 +23,7 @@ import { AppState, AppStateStatus } from 'react-native'
 import { remoteConfigService } from 'src/services/remote-config'
 import { locale } from 'src/helpers/locale'
 import { pushNotifcationRegistration } from 'src/push-notifications/push-notifications'
+import { useApiUrl } from './use-settings'
```

Comment:
Is this returning code api url (when set in dev menu)?

Reply:
Yes it does